### PR TITLE
fix(build): fix source-map and uglify

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
This PR fixes two things:
1. we don't have to specify `-p` if we have `UglifyJsPlugin` inside the `webpack.config.js`
2. fix the `source-map` issue with es6-promises package mentioned [here](https://github.com/contentful/contentful.js/issues/145)

References : 
Webpack docs
>If --devtool source-map is omitted, the output still contains the sourcemap URI, but in an invalid position resulting in the browser ignoring it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -153,3 +153,4 @@ $RECYCLE.BIN/
 *.lnk
 
 # End of https://www.gitignore.io/api/node,windows,osx,linux,vim
+*.patch
```
Comment (axe312ger) [inline .gitignore]: Please add to beginning of the file as the custom entries

File: webpack.config.js [hunk 2]
```diff
@@ -58,6 +59,7 @@ module.exports = [
     module: {
       loaders
     },
+    devtool: 'source-map',
```
Comment (axe312ger) [inline webpack.config.js]: For performance reasons, I would prefer `cheap-module-source-map` --> https://webpack.js.org/configuration/devtool/#devtool

**Comment by Khaledgarbaya** [comment]
@axe312ger can you take a look again ?

**Comment by aChudinov** [comment]
@Khaledgarbaya 
After this PR we have an issue with webpack UglifyJs
```
Error: app-bb3e1f83d765aff08857.js from UglifyJs
Unexpected token: operator (>) [./~/contentful/dist/contentful.js:1502,0][app-bb3e1f83d765aff08857.js:192762,17]
```

As I see, now we have ES6 code in compiled `dist` folder, like
```
const linkGetter = __WEBPACK_IMPORTED_MODULE_1_mem___default()(getLinksFromIncludes, {
  cacheKey: a => a.sys.id
});
```

**Pull request merged**